packages/twenty-server/src/engine/core-modules/file/services/file.service.ts

```diff
@@ -48,4 +48,29 @@ export class FileService {
 
     return signedPayload;
   }
+
+  async deleteFile({
+    folderPath,
+    filename,
+    workspaceId,
+  }: {
+    folderPath: string;
+    filename: string;
+    workspaceId: string;
+  }) {
+    const workspaceFolderPath = `workspace-${workspaceId}/${folderPath}`;
+
+    return await this.fileStorageService.delete({
```

Comment:
On my side it didn't delete the files locally? I still have them in my .local-storage folder. I can see the job is processed in the queue but the file still seems to be there. No error in the queue log. Any idea why?

Reply:
Did you try with both drivers (S3+Local)? Or just one?

Reply:
Please ignore above comment, it did work after I restarted the queue...